### ISSUE TITLE
Constant fatal error: "Encountered an error during download..."

### ISSUE THREAD

**Issue by SunGod** [civitai/sd_civitai_extension#112]
For the last few days I've been getting this error constantly when trying to download anything through the CivitAI extension. I've confirmed it's not a connectivity issue, not a local network issue, not an available filesystem space issue, not an issue of filesystem permissions, not an issue of CivitAI being down, etc. This pretty much started unexpectedly, and best I can tell it began after one or two of the recent CivitAI extension updates for A1111 occured. Now I basically can't download anything through the extension, its effectively broken. See image below for example issue. Doesn't matter if it's a LORA, checkpoint, pose or anything, seems to happen 100% of the time, time of day seems irrelevant so it doesn't seem to be a CivitAI traffic issue. :-(

My setup: 
•  version: v1.6.0 
•  python: 3.10.6  
•  torch: 2.0.1+cu118  
•  xformers: 0.0.20  
•  gradio: 3.41.2  
•  checkpoint: ec41bd2a82

![image](https://github.com/civitai/sd_civitai_extension/assets/2029061/f114d632-7a0e-4f61-9ae7-a03f0210cea3)

Steps to reproduce:
1. Select CivitAI tab
2. Search for any checkpoint, LORA, pose, whatever. Doesn't seem to matter.
3. Select one you want to try and download.
4. Click Download Model button.
5. Extension will try to download model, then the message shown in the example photo will appear. 

Note that for ALL the items you try to download, the default "File" name will be "None SafeTensor None <filesize MB>". I've never noticed the "None" words appear in this field until recently. Not sure if it's related to this issue or not. Sometimes instead of "SafeTensor" it says "Other".

**Comment by SunGod** [comment]
Update 10/2/2023: 100% its the extension. Confirmed it's not VPN, am able to directly DL any assets from Civit AI in another tab on the same machine using the same browser. Additionally, when I do manually download (for example) a LORA directly from Civit AI without using the extension, then come back to the extension and search for the same LORA I just manually installed, I'm able to use "Save Images" within the extension just fine and it works no problem. 

Basically the extensions' "Download Model" function is completely broken right now. :-(
